azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/configuration/CallConfiguration.kt

```diff
@@ -5,13 +5,16 @@ package com.azure.android.communication.ui.calling.configuration
 
 import com.azure.android.communication.common.CommunicationTokenCredential
 import com.azure.android.communication.ui.calling.DiagnosticConfig
+import com.azure.android.communication.ui.calling.models.CallCompositePushNotificationInfo
 import com.azure.android.communication.ui.calling.models.CallCompositeParticipantRole
 import java.util.UUID
 
 internal enum class CallType {
     GROUP_CALL,
     TEAMS_MEETING,
     ROOMS_CALL,
+    ONE_TO_N_CALL_OUTGOING,
```

Comment:
might be ok as just 
CALL_OUTGOING
and
CALL_INCOMING

Not sure if ONE_TO_N_CALL is necessary in the call type.